src/composables/transactionList.ts

```diff
@@ -1,30 +1,33 @@
-import { orderBy, uniqBy } from 'lodash-es';
-import { ref, watch } from 'vue';
-import { Encoded, Tag } from '@aeternity/aepp-sdk';
+import { computed, ref, watch } from 'vue';
+import { Encoded } from '@aeternity/aepp-sdk';
 import type {
   ITransaction,
   ITransactionsState,
   IAccountTransactionsState,
   IDefaultComposableOptions,
+  Protocol,
 } from '@/types';
 import {
   TRANSACTIONS_LOCAL_STORAGE_KEY,
   TX_DIRECTION,
+  TXS_PER_PAGE,
 } from '@/constants';
 import {
-  fetchJson,
   getLocalStorageItem,
   setLocalStorageItem,
 } from '@/utils';
-import JsonBig from '@/lib/json-big';
-import { AEX9_TRANSFER_EVENT } from '@/protocols/aeternity/config';
-import { useAeNetworkSettings } from '@/protocols/aeternity/composables';
 
+import { ProtocolAdapterFactory } from '@/lib/ProtocolAdapterFactory';
+import { AE_MDW_TO_NODE_APPROX_DELAY_TIME, AEX9_TRANSFER_EVENT } from '@/protocols/aeternity/config';
+import { useMiddleware } from '@/composables/middleware';
+import { isEqual, orderBy, uniqBy } from 'lodash-es';
+import { useBalances } from '@/composables/balances';
+import { createNetworkWatcher } from '@/composables/networks';
```

Comment:
I don;t understand why you changed the paths here. This is local import from the same directory. Please revert this change.

Reply:
Dependency cycle detected.

Reply:
Its not related to dependency cycle. It's a matter of the path:
```suggestion
import { useBalances } from './balances';
import { createNetworkWatcher } from './networks';
```